packages/toolkit/scripts/project/build.js

```diff
@@ -0,0 +1,45 @@
+const { execSync } = require('child_process');
+const chalk = require('chalk');
+
+const { log } = console;
+
+const fs = require('fs');
+const { getProjectRoot, getProjectVariables, setEnvVariables } = require('../../utils');
+
+const description = '10up-toolkit project build';
+
+const run = async () => {
+	const root = getProjectRoot();
+
+	if (!root) {
+		log(chalk.red('This is not a project.'));
+		process.exit(1);
+	}
+
+	const variables = getProjectVariables();
+
+	if (!variables) {
+		log(chalk.red('No .tenup.yml found.'));
+		process.exit(1);
+	}
+
+	// Loop through variables and set them as environment variables
+	Object.keys(variables).forEach((key) => {
+		process.env[key] = variables[key];
+	});
+
+	setEnvVariables(variables);
```

Comment:
What's the difference between lines 27-29 and this function?

Reply:
Good catch. Fixed now.